content/lxc/getting-started.md

```diff
@@ -187,6 +187,15 @@ If we wait about 5 seconds and check again, then the container does have an IP a
     NAME        STATE   AUTOSTART GROUPS IPV4       IPV6 UNPRIVILEGED
     mycontainer RUNNING 1         -      10.0.3.152 -    false
 
+If the container does not have an IP address, we may need to [configure the firewall](https://documentation.ubuntu.com/lxd/en/latest/howto/network_bridge_firewalld/#how-to-configure-your-firewall). For example, on Ubuntu 22.04
```

Comment:
Thank you, @stgraber. I intend feebd403248e134a959d7fa75f9382d1095c952d links to the url you provide instead of the `documentation.ubuntu.com` url the previous commit used.